CHANGES.rst

```diff
@@ -5,8 +5,23 @@ grafanimate changelog
 
 in progress
 ===========
-- Add Dockerfile, and publish OCI images to GHCR. Thanks, @intermittentnrg.
-
+- OCI: Added Dockerfile, and started publishing OCI images to GHCR
+- OCI: Dockerfile: Added ``xauth`` and ``ffmpeg`` packages
+- OCI: Dockerfile: Use ``tini`` for bootstrapping ``xvfb-run``
+- Improved finding executable to ``firefox-esr``. Thanks, @gogglespisano.
+- Improved support with ``AnimationSequence.recurrence``
+- Migrated to ``pyproject.toml``
+- Added support for Python up to version 3.13, and dropped support for
+  Python up to version 3.8.
+- Added support for recent versions of Grafana based on React.js
+- Grafana Studio: Update title in solo panel mode, improve login method,
```

Comment:
We also dropped support for grafana without scenes feature

Reply:
Thanks. I've added a relevant item to the change log.